config/config.go

```diff
@@ -64,6 +64,8 @@ type Configuration struct {
 	AccountRequired bool `mapstructure:"account_required"`
 	// Local private file containing SSL certificates
 	PemCertsFile string `mapstructure:"certificates_file"`
+	// Custom headers set by Nginx to handle request timeout in queue or any other custom headers
+	CustomHeaders CustomHeaders `mapstructure:"custom_headers"`
```

Comment:
You also need to add `v.SetDefault(...)` values in `SetupViper` for environmental overrides to be considered.

Reply:
Added